pkg/agent/types/multicast.go

```diff
@@ -36,8 +36,7 @@ var (
 )
 
 type MulticastValidator interface {
-	// Validate checks whether IGMP report from Pod(podNamespace/podName) to groupAddress should be dropped,
+	// GetMcastNPRuleInfo checks whether IGMP report from Pod(podNamespace/podName) to groupAddress should be dropped,
 	// and returns multicast NetworkPolicy information.
-	// TODO: refacor the function name and return type here
-	Validate(podname, podNamespace string, groupAddress net.IP, igmpType uint8) (McastNPValidationItem, error)
+	GetMcastNPRuleInfo(podname, podNamespace string, groupAddress net.IP, igmpType uint8) (McastNPRuleInfo, error)
```

Comment:
And GetIGMPRuleInfo() or QueryIGMPRuleInfo()?

Reply:
done